packages/@aws-cdk/aws-s3/README.md

```diff
@@ -616,3 +616,41 @@ const bucket = new s3.Bucket(this, 'MyBucket', {
   }]
 });
 ```
+
+## Object Lock Configuration
+
+[Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock-overview.html)
+can be configured to enable a write-once-read-many model for an S3 bucket. Object Lock must be
+configured when a bucket is created; if a bucket is created without Object Lock, it cannot be
+enabled later via the CDK.
+
+Object Lock can be enabled on an S3 bucket by specifying:
+
+```ts
+const bucket = new s3.Bucket(this, 'MyBucket', {
+  objectLock: {
+    enabled: true
+  }
+});
```

Comment:
Is this valid? `parseObjectLockConfig()` doesn't specify default retention period or mode, and the API docs suggest that both are required if `enabled` is true.